x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/support/StringMatcherTests.java

```diff
@@ -30,16 +30,12 @@ public void testEmptySet() throws Exception {
     public void testMatchAllWildcard() throws Exception {
         Supplier<String> randomPattern = () -> {
             final String s = randomAlphaOfLengthBetween(3, 5);
-            switch (randomIntBetween(1, 4)) {
-                case 1:
-                    return s;
-                case 2:
-                    return s + "*";
-                case 3:
-                    return "*" + s;
-                default:
-                    return "*" + s + "*";
-            }
+            return switch (randomIntBetween(1, 4)) {
+                case 1 -> s;
+                case 2 -> s + "*";
+                case 3 -> "*" + s;
+                default -> "*" + s + "*";
```

Comment:
I can't decide if this bothers me or not. Everywhere else we do something like the following. It's up to you if it bothers you too.

```suggestion
                case 4 -> "*" + s + "*";
                default -> throw new AssertionError();
```